### ISSUE TITLE
Post Cache Dependencies Step Failing

### ISSUE THREAD

**Issue by rishabprasad005** [actions/cache#547]
The Post Cache step is failing with error message:-

```
Post job cleanup.
/bin/tar -cz -f /home/runner/work/_temp/d8a6b35c-f910-4b09-aeb6-4c176c24219a/cache.tgz -C /home/runner/.m2/repository .
Warning: read ECONNRESET
events.js:187
      throw er; // Unhandled 'error' event
      ^

Error: ESPIPE: invalid seek, read
Emitted 'error' event on ReadStream instance at:
    at internal/fs/streams.js:167:12
    at FSReqCallback.wrapper [as oncomplete] (fs.js:470:5) {
  errno: -29,
  code: 'ESPIPE',
  syscall: 'read'
}
```
Full Logs: https://github.com/Apicurio/apicurio-registry/pull/1235/checks?check_run_id=1934916176

**Comment by DenWav** [comment]
Duplicate of #467

**Comment by dhadka** [comment]
Merged #596 to fix this issue by catching and logging warnings.  The save step should no longer fail.  Fix in `v2.1.6`.
